src/entities/priceImpact/index.ts

```diff
@@ -45,4 +54,134 @@ export class PriceImpact {
         const priceImpact = (amountInitial - amountFinal) / amountInitial / 2;
         return PriceImpactAmount.fromDecimal(`${priceImpact}`);
     };
+
+    static addLiquidityUnbalanced = async (
+        input: AddLiquidityUnbalancedInput,
+        poolState: PoolStateInput,
+    ): Promise<PriceImpactAmount> => {
+        // inputs are being validated within AddLiquidity
+
+        // simulate adding liquidity to get amounts in
+        const addLiquidity = new AddLiquidity();
+        const { amountsIn, bptOut } = await addLiquidity.query(
+            input,
+            poolState,
+        );
+        const poolTokens = amountsIn.map((a) => a.token);
+
+        // simulate removing liquidity to get amounts out
+        const removeLiquidity = new RemoveLiquidity();
+        const removeLiquidityInput: RemoveLiquidityInput = {
+            chainId: input.chainId,
+            rpcUrl: input.rpcUrl,
+            bptIn: bptOut.toInputAmount(),
+            kind: RemoveLiquidityKind.Proportional,
+        };
+        const { amountsOut } = await removeLiquidity.query(
+            removeLiquidityInput,
+            poolState,
+        );
+
+        // diffs between unbalanced and proportional amounts
+        const diffs = amountsOut.map((a, i) => a.amount - amountsIn[i].amount);
```

Comment:
In some smart contracts, as the Vault, and in the b-sdk test helpers is used the word "deltas" instead of "diffs", maybe it would be better to use "deltas" to keep the pattern